src/api/firebase/firebase.types.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
```

Comment:
In my opinion we should increase `max-len` property to 100 or 120 characters.

Reply:
ESLint _max-len_ and Prettier _printWidth_ are matching now. Both set to 120.